src/reflect/mima-filters/2.12.0.forwards.excludes

```diff
@@ -44,6 +44,8 @@ ProblemFilters.exclude[IncompatibleMethTypeProblem]("scala.reflect.io.ZipArchive
 ProblemFilters.exclude[IncompatibleResultTypeProblem]("scala.reflect.io.FileZipArchive.allDirs")
 ProblemFilters.exclude[DirectMissingMethodProblem]("scala.reflect.runtime.SynchronizedSymbols#SynchronizedSymbol.typeConstructor")
 
+ProblemFilters.exclude[DirectMissingMethodProblem]("scala.collection.TraversableView.noBuilder")
```

Comment:
_pace_ MiMa, this might need to be `scala.collection.TraversableView$.noBuilder`.

Reply:
It's actually just in the wrong file, should be in `src/library/...`. I'll push a fix.